spec/requests/physical_storages_spec.rb

```diff
@@ -21,6 +21,44 @@
     end
   end
 
+  context "Delete physical storage" do
+    it "Deletion of a single Physical Storage" do
+      provider = FactoryBot.create(:ems_autosde, :name => 'Autosde')
+      physical_storage = FactoryBot.create("ManageIQ::Providers::Autosde::StorageManager::PhysicalStorage", :name => 'test_storage', :ext_management_system => provider)
+      api_basic_authorize('physical_storage_delete')
+
+      post(api_physical_storage_url(nil, physical_storage), :params => gen_request(:delete))
+
+      expect_single_action_result(:success => true, :message => /Detaching Physical Storage id:#{physical_storage.id}/i, :href => api_physical_storage_url(nil, physical_storage))
+    end
+
```

Comment:
Can you add specs for some of the error handling?  E.g. PhysicalStorage that doesn't support `:delete`, an ID that doesn't exist returns a 404, etc...

Reply:
done